providers/os/id/networki/windows_routes.go

```diff
@@ -0,0 +1,387 @@
+// Copyright (c) Mondoo, Inc.
```

Comment:
naming of the files is inconsistent, darwin is different from linux and windows. I also think the windows routes file is not consistent with the native implementation. So I think this should end with `_pwsh.go` or something similar.